express/code/blocks/ribbon-banner/ribbon-banner.css

```diff
@@ -1,54 +1,84 @@
 .section .ribbon-banner {
+    /* color variables for easy theming and fewer duplicate rules */
+    --rb-text-color: var(--Alias-content-typography-Heading, #131313);
```

Comment:
Do we need intermediary variables here?

--rb-cta-color: var(--color-gray-900); 

How about just using the token directly:` ---color-gray-900)`